jest-setup.js

```diff
@@ -1,2 +1,32 @@
 // Jest setup provided by Grafana scaffolding
 import './.config/jest-setup';
+// Used by useMeasure
```

Comment:
is this in some library code? i couldn't find any usage of `useMeasure` inside sitewise

Reply:
Copied it from grafana main and forgot to add the proper comment, smh. Fixed!